app/assets/javascripts/Components/ContentListView/ContentListView.tsx

```diff
@@ -156,9 +170,9 @@ const ContentListView: FunctionComponent<Props> = ({ application, appState }) =>
               <div className="sk-h2 font-semibold title">{panelTitle}</div>
               <button
                 className="sk-button contrast wide"
-                title="Create a new note in the selected tag"
-                aria-label="Create new note"
-                onClick={() => createNewNote()}
+                title={isFilesSmartView ? 'Upload file' : 'Create a new note in the selected tag'}
+                aria-label={isFilesSmartView ? 'Upload file' : 'Create new note'}
```

Comment:
Shouldn't `title` and `aria-label` be same for `!isFilesSmartView` case?

Reply:
Yeah, makes sense.